packages/repmgr13/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV TERM=xterm \
+    RPM_SPEC=repmgr.spec
+
+RUN ARCH=$(uname -m) && \
+    # TODO figure out s390x
+    dnf -y remove *subscription-manager* && \
+    dnf -y update && \
+    dnf -y install \
+      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
+      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \
+      https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    dnf -y module enable postgresql:13 && \
+    dnf -y group install --disablerepo=ubi-8-appstream-rpms "development tools" && \
```

Comment:
The disablerepo is described in the commit message, copied here for convenience:

    Disable ubi-8-appstream while installing dev tools for now

    Currently, the versions of some of the dev tools in ubi-8-appstream,
    specifically rpmbuild conflict with what's available in related dependencies in
    centos appstream 8.

    rpm-build-4.14.3-23.el8.x86_64 : Scripts and executable programs used to build packages
    Repo        : appstream
    Matched from:
    Filename    : /usr/bin/rpmbuild

    rpm-build-4.14.3-24.el8_6.x86_64 : Scripts and executable programs used to build packages
    Repo        : ubi-8-appstream-rpms
    Matched from:
    Filename    : /usr/bin/rpmbuild

    rpm-build-4.14.3-24.el8_6.x86_64 was pushed today according to https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/appstream/os/Packages/r/
    so we'll just disable that repo for installing the dev tools group while they
    fix the issue.